extensions/resource-deployment/src/interfaces.ts

```diff
@@ -202,22 +209,42 @@ export enum ToolType {
 	Azdata
 }
 
+export const enum ToolStatus {
+	NotInstalled = 'Not Installed',
+	Installed = 'Installed',
+	Installing = 'Installing ...',
```

Comment:
>Installing ... [](start = 15, length = 14)

Not sure why you have what appears to be a display string here. As mentioned in the other comment you should have the localized strings as declared constants somewhere (not here likely)

Reply:
Changed to not be display strings